README.md

```diff
@@ -1 +1,254 @@
-# Microverse-Portfolio-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse Portfolio Project</b></h3>
+
+</div>
```

Comment:
-  Please remove the template logo, if you have your own logo you can replace it with it one, if not please delete it :100: